aws/logs_monitoring/lambda_function.py

```diff
@@ -515,6 +529,10 @@ def add_metadata_to_lambda_log(event):
     tags = list(set(tags))
     tags.sort()  # Keep order deterministic
 
+    # Set custom tags on DD_FORWARDER_TELEMETRY
+    DD_FORWARDER_TELEMETRY_TAGS.extend(custom_lambda_tags)
```

Comment:
I don't expect us to add lambda tags to the forwarder, it would dramatically increase the cardinality.